README.md

```diff
@@ -20,3 +20,22 @@ It is easy to install the GRACC Request Daemon with virtualenv
 A docker image with gracc-request installed in available as opensciencegrid/gracc-request.  
 
 
+## Corrections
+
+The configuration for a correction is
+
+    [[Corrections]]
+    index = 'gracc.corrections'
+    doc_type = 'project'
+    match_fields = ['ProjectName']
+    source_field = 'CorrectedProjectName'
+    dest_field = 'ProjectName'
```

Comment:
Can you put in an example?  Both the ElasticSearch document and the configuration file.